uaclient/cli.py

```diff
@@ -1579,6 +1631,37 @@ def action_refresh(args, *, cfg: config.UAConfig):
     return 0
 
 
+def configure_apt_proxy(
+    cfg: config.UAConfig, scope: AptProxyScope, args: Dict
+) -> None:
+    """
+    Handles setting part the apt proxies - global and uaclient scoped proxies
+    """
+    # needs to import from within here or else unit tests fail
+    # due to wrong importation path issues i.e
+    # tests will use cli.setup_apt_proxy over apt.setup_apt_proxy
+
+    set_value = args.get("set_value")
+    set_key = args.get("set_key")
+    if scope == AptProxyScope.GLOBAL:
+        kwargs = {
+            "http_proxy": cfg.global_apt_http_proxy,
+            "https_proxy": cfg.global_apt_https_proxy,
+            "proxy_scope": scope,
+        }
+    elif scope == AptProxyScope.UACLIENT:
+        kwargs = {
+            "http_proxy": cfg.ua_apt_http_proxy,
+            "https_proxy": cfg.ua_apt_https_proxy,
+            "proxy_scope": scope,
+        }
+    if "https" in set_key:
+        kwargs["https_proxy"] = set_value
+    else:
+        kwargs["http_proxy"] = set_value
+    setup_apt_proxy(**kwargs)
```

Comment:
Making the variables explicit instead of passing a `kwargs` object here makes it more readable, and Mypy stops complaining as well 

```suggestion
    if scope == AptProxyScope.GLOBAL:
        http_proxy = cfg.global_apt_http_proxy
        https_proxy = cfg.global_apt_https_proxy
    elif scope == AptProxyScope.UACLIENT:
        http_proxy = cfg.ua_apt_http_proxy
        https_proxy = cfg.ua_apt_https_proxy
    if "https" in set_key:
        https_proxy = set_value
    else:
        http_proxy = set_value
    setup_apt_proxy(http_proxy, https_proxy, scope)
```